integ-test/src/test/java/org/opensearch/sql/calcite/clickbench/CalcitePPLClickBenchIT.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.calcite.clickbench;
+
+import java.util.Set;
+import org.junit.FixMethodOrder;
+import org.junit.runners.MethodSorters;
+
+@FixMethodOrder(MethodSorters.JVM)
+public class CalcitePPLClickBenchIT extends PPLClickBenchIT {
+
+  @Override
+  public void init() throws Exception {
+    super.init();
+    enableCalcite();
+    disallowCalciteFallback();
+  }
+
+  @Override
+  protected Set<Integer> ignored() {
+    return Set.of(29, 41, 43);
```

Comment:
q29: REGEXP_REPLACE() is not supported.
q41: Failed in Calcite engine, TBD
q43: DATE_TRUNC() is not supported.